ecommerce/backend/API/views/address.py

```diff
@@ -0,0 +1,65 @@
+from API.models.user import Address
+from rest_framework import exceptions
+from rest_framework.response import Response
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework import status
+
+from ..utils import permissions, Role
+from ..models import User, Address
+from ..serializers.address_serializer import *
+
+@api_view(['GET', 'DELETE'])
+@permission_classes([permissions.AllowAnonymous])
+def manage_single_address(request, customer_id, address_id):
+    if User.objects.filter(id=customer_id).first() is None:
+        return Response(status=status.HTTP_400_BAD_REQUEST)
+    #get single address
+    if request.method == 'GET':
+        address = Address.objects.filter(user_id=customer_id).filter(id=address_id)
+        if address is None:
+            return Response({'successful': False, 'message': "No such address is found"})
+        address_serializer = AddressResponseSerializer(address, many=True)
+        return Response(address_serializer.data)
+    #delete single address 
+    elif request.method == 'DELETE':
```

Comment:
At delete, you are deleting address pyhsicially from database. Later it may cause errors because of foreign key constraints. For example, I order something to addres with id 1. Then at purchases table my order references address with id 1. We should apply soft delete by adding an addğitional flag column. 

For now, it is okay. I will approve it. Later we can change that.